src/components/tile-implementations/weather-alerts/useWeatherAlertsApi.test.tsx

```
@@ -0,0 +1,54 @@
+  <MockDataServicesProvider
+    setup={({ mapperRegistry }) => {
+      mapperRegistry.register(TileType.WEATHER_ALERTS, new WeatherAlertsDataMapper());
+    }}
+  >
+    {children}
+  </MockDataServicesProvider>
+);
+
+describe('useWeatherAlertsApi', () => {
+  it('fetches and maps weather alerts data successfully', async () => {
+    (globalThis.fetch as unknown as { mockResolvedValueOnce: Function }).mockResolvedValueOnce({
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)